src/Raven.Server/Documents/Handlers/Debugging/ThreadsHandler.cs

```diff
@@ -13,6 +14,32 @@ namespace Raven.Server.Documents.Handlers.Debugging
 {
     public class ThreadsHandler : RequestHandler
     {
+        [RavenAction("/admin/debug/threads/top", "GET", AuthorizationStatus.Operator, IsDebugInformationEndpoint = true)]
```

Comment:
Isn't this just runaway?

Reply:
There we are using `thread.TotalProcessorTime.TotalMilliseconds`

Reply:
It doesn't provide the actual CPU usage. @karmeli87 maybe we can combine the 2 endpoints?

Reply:
I think those are 2 different measures. `runaway` measuring from the start-up, while `top` is only for the last second.

But we can merge them if you like, the question is how we will order them?

Reply:
Reminder - @grisha-kotler ?

Reply:
Merged both endpoints